src/main/java/command/AddBMICommand.java

```diff
@@ -0,0 +1,20 @@
+package command;
+
+import activeedge.userdetails.LogBMI;
+import static activeedge.userdetails.UserDetailsList.DETAILS_LIST;
```

Comment:
can you change `DETAILS_LIST` to `detailsList`? We should only use capitals for constants.